docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md

```diff
@@ -54,6 +120,7 @@ Make sure that the following changes have been applied in transfer objects:
 | TRANSFER  | TYPE  | EVENT | PATH  |
 |-|-|-|-|
 | MerchantProductTableCriteria | class | Created | src/Generated/Shared/Transfer/MerchantProductTableCriteriaTransfer |
+| ProductAbstract.idMerchant | attribute | Created | src/Generated/Shared/Transfer/ProductAbstractTransfer |
```

Comment:
already exists in MerchantProduct module (Merchant product feature which we depend on) so this property will not appear, can be removed